rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/NodeSource.java

```diff
@@ -145,7 +174,7 @@ public String getIcon() {
         /** true if one of the contained nodes contains 'VIRT' in its URL */
         private boolean virtual = false;
 
-        Host(String hostName, String sourceName) {
+        public Host(String hostName, String sourceName) {
             this.hostName = hostName;
             this.nodes = new HashMap<String, Node>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)